lib/palette.js

```diff
@@ -98,6 +99,35 @@ function map_palette(a, b, type, bc)
 }
 
 /**
+* Returns the closest (or furthest) color to targetColor in palette, operating in the L,a,b colorspace for performance
+* @param {labcolor} targetColor should have fields L,a,b
+* @param [{labcolor}] palette each element should have fields L,a,b
+* @param 'findFurthest' should be falsy to find the closest color
+* @return {labcolor}
+*/
+function palette_lab(targetColor, palette, findFurthest)
```

Comment:
I would prefer if this was implemented like map_palette (a map_lab_palette) instead and then implement the closest/furthest function as it is implemented for RGB(A) in index.js. This should not give much performance overhead and make it possible to use for mapping multiple colors, which is useful. Preferably the function arguments should be similar to map_palette (minus the background color)